pyproject.toml

```diff
@@ -17,8 +17,12 @@ dependencies = [
     "pydantic>=2.0.0",
     "tiktoken>=0.5.0",
     "httpx>=0.24.0",
+    "groq>=0.4.0",
```

Comment:
I think we need to add to requirements.txt too

Reply:
solid, added:
3c38da6e7ae61198457a74e52a9d4abb1c2be9ab